server/services/offendersService.js

```diff
@@ -691,7 +695,11 @@ module.exports = function createOffendersService(
         const dbRecord = await formService.getCategorisationRecord(raw.bookingId, transactionalDbClient)
         const pomData = pomMap.get(nomisRecord.offenderNo)
 
-        if (isInitialInProgress(dbRecord) || isNextReviewAfterRelease(nomisRecord, releaseDateMap.get(raw.bookingId))) {
+        if (
+          isInitialInProgress(dbRecord) ||
+          (!isAwaitingApproval(dbRecord.status) &&
+            isNextReviewAfterRelease(nomisRecord, releaseDateMap.get(raw.bookingId)))
+        ) {
```

Comment:
This is the only actual behavioural change in this PR. 

I still think this is overly complex - however, confidence is not especially high despite the seemingly high coverage here.

Nothing failed when I made this change, either in Groovy or in Jest. 

None of this was covered in Jest. 

Before:

```
------------------------------------------------|---------|----------|---------|---------|
File                                            | % Stmts | % Branch | % Funcs | % Lines | 
------------------------------------------------|---------|----------|---------|---------|
All files                                       |   73.72 |    70.24 |   69.87 |   73.75 |
  offendersService.js                           |   75.69 |    81.73 |    80.7 |   75.46 |
  utils.js                                      |   85.03 |    71.21 |   85.36 |   85.18 |
```

After:

```
------------------------------------------------|---------|----------|---------|---------|
File                                            | % Stmts | % Branch | % Funcs | % Lines | 
------------------------------------------------|---------|----------|---------|---------|
All files                                       |   73.92 |    70.89 |   69.92 |   73.95 |
  offendersService.js                           |   76.66 |    84.07 |   80.81 |    76.5 |
  utils.js                                      |   85.03 |    72.72 |   85.36 |   85.18 |
```